src/khoj/search_type/image_search.py

```diff
@@ -26,9 +25,6 @@
 
 
 def initialize_model(search_config: ImageSearchConfig):
```

Comment:
We should really just delete this until we build better support for this feature 🙈

Reply:
Yeah, we really should!